examples/interop/ios-compose-in-swiftui/shared/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+@file:Suppress("OPT_IN_IS_NOT_ENABLED")
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+}
+
+version = "1.0-SNAPSHOT"
+
+kotlin {
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
+    ).forEach { iosTarget ->
+        iosTarget.binaries {
+            framework {
+                baseName = "shared"
+                isStatic = true
+            }
+        }
+    }
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(compose.ui)
+                implementation(compose.foundation)
+                implementation(compose.material)
+                @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
+                implementation(compose.components.resources)
+            }
+        }
+        val iosMain by creating {
+            dependsOn(commonMain)
+        }
+        val iosX64Main by getting {
+            dependsOn(iosMain)
+        }
+        val iosArm64Main by getting {
```

Comment:
Why do we need explicitly mention X64/Arm64 here?

Reply:
Because ios() configuration doesn't provide it

Reply:
Sorry, correct to say: 
ios() confguration doesn't provide iosSimulatorArm64 target.
We can't use ios(), so we are using all targets instead.